Index: requirements.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>gunicorn==20.1.0\r\ncertifi==2020.6.20\r\nchardet==3.0.4\r\nclick==7.1.2\r\ndominate==2.5.2\r\nFlask==1.1.2\r\nFlask-Bootstrap==3.3.7.1\r\nFlask-CKEditor==0.4.4.1\r\nFlask-Gravatar==0.5.0\r\nFlask-Login==0.5.0\r\nFlask-SQLAlchemy==2.4.4\r\nFlask-WTF==0.14.3\r\nidna==2.10\r\nitsdangerous==1.1.0\r\nJinja2==2.11.2\r\nMarkupSafe==1.1.1\r\nrequests==2.24.0\r\nSQLAlchemy==1.3.19\r\nurllib3==1.25.10\r\nvisitor==0.1.3\r\nWerkzeug==1.0.1\r\nWTForms==2.3.3\r\nemail-validator==1.1.2\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/requirements.txt b/requirements.txt
--- a/requirements.txt	(revision 255d66dc2de4de9a0340d2bd67f90e1ce8b71cc3)
+++ b/requirements.txt	(date 1645379802770)
@@ -20,4 +20,4 @@
 visitor==0.1.3
 Werkzeug==1.0.1
 WTForms==2.3.3
-email-validator==1.1.2
+email-validator==1.1.3
